.github/composite_actions/log_cw_metric_wrapper/action.yaml

```diff
@@ -83,7 +75,7 @@ runs:
         job-status: ${{ inputs.job-status }}
         github-token: ${{ inputs.github-token }}
 
-        metric-name: "github_metric_2.0"
```

Comment:
If this is not a major change why not use 2.1 or 2.01 instead?

Reply:
Kind of wanted to use a whole integer bump until this is stabilized and our dashboards are solidified.

Reply:
Right so if there are more changes incoming shouldn't we stay at 2 and then bump to a major version 3 when things are stabilized?

Reply:
Fixed